pkg/shortnames/shortnames_test.go

```diff
@@ -0,0 +1,100 @@
+package shortnames
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
+)
+
+func TestIsShortName(t *testing.T) {
+	tests := []struct {
+		input       string
+		isShortName bool
+		mustFail    bool
+	}{
+		// SHORT NAMES
+		{
```

Comment:
(Could each `{…},` be condensed to one line? That would mean less scrolling and make it easier to check what cases are covered.)